package.json

```diff
@@ -64,6 +64,7 @@
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
     "start": "./bin/start.js",
     "pretest": "standard",
-    "test": "nyc tap --no-cov './test/{unit,integration}/**/*-test.js'"
+    "test": "nyc tap --no-cov './test/{unit,integration}/**/*-test.js'",
+    "postinstall": "node ./set-start.js"
```

Comment:
I've been experimenting with this by installing packages in different ways and everything seems to be working the way its supposed to. Though of course that doesn't mean it won't ever break. So I'd like to add a check like you said.